openreview/arr/management/setup_reviewer_matching.py

```diff
@@ -270,13 +270,21 @@ def replace_edge(existing_edge=None, edge_inv=None, new_weight=None, submission_
             previous_submission = client_v1.get_note(previous_id)
             previous_venue_id = previous_submission.invitation.split('/-/')[0]
             previous_parent_reviewers = client_v1.get_group(f"{previous_venue_id}/Paper{previous_submission.number}/Reviewers")
-            previous_reviewers = client_v1.get_group(f"{previous_venue_id}/Paper{previous_submission.number}/Reviewers/Submitted")
+            previous_reviewers = openreview.tools.get_group(client_v1, f"{previous_venue_id}/Paper{previous_submission.number}/Reviewers/Submitted")
+            if previous_reviewers is None:
+                print(f"no previous reviewers for {submission.id}")
+                continue
```

Comment:
Nothing else needs to run after this is the Reviewers/Submitted group does not exist?

Reply:
Actually yeah I should still give access to the previous submission to avoid confusion for the new reviewers - thanks!